{"name": "test_companyRegister", "status": "broken", "statusDetails": {"message": "pymysql.err.ProgrammingError: (1146, \"Table 'chengtay-zero-copy.zero_user_auth' doesn't exist\")", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"ÊµãËØï http://10.10.128.152:10053/user/register ‰∏≠Â∞èÂæÆ‰ºÅ‰∏öÊ≥®ÂÜåÊµÅÁ®ã\")\n    @allure.testcase(\"http://10.10.128.152:10053/user/register\", \"Ê≥®ÂÜå üëá\")\n    def test_companyRegister():\n        if runMode == 'UI':\n            def_name = sys._getframe().f_code.co_name\n            test_Assert = Assert.Assertions(def_name)\n            logger.info(\"ÂºÄÂßãÊâßË°åËÑöÊú¨%s:\\n\", def_name)\n    \n            driver = webdriver.Chrome()\n            driver.maximize_window()\n            driver.get(url_ui_register)\n            # db‰∏≠Ê∏ÖÈô§Â∑≤Ê≥®ÂÜåÁöÑË¥¶Êà∑\n>           deleteInforMobile()\n\ntest_flow\\test_Authentication\\test_login.py:62: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ncommon\\dbLink.py:114: in deleteInforMobile\n    Sqldata(sqlStr14, 2)\ncommon\\dbLink.py:29: in Sqldata\n    cursor.execute(sqlStr)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\cursors.py:163: in execute\n    result = self._query(query)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\cursors.py:321: in _query\n    conn.query(q)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\connections.py:505: in query\n    self._affected_rows = self._read_query_result(unbuffered=unbuffered)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\connections.py:724: in _read_query_result\n    result.read()\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\connections.py:1069: in read\n    first_packet = self.connection._read_packet()\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\connections.py:676: in _read_packet\n    packet.raise_for_error()\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\protocol.py:223: in raise_for_error\n    err.raise_mysql_exception(self._data)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ndata = b\"\\xffz\\x04#42S02Table 'chengtay-zero-copy.zero_user_auth' doesn't exist\"\n\n    def raise_mysql_exception(data):\n        errno = struct.unpack('<h', data[1:3])[0]\n        errval = data[9:].decode('utf-8', 'replace')\n        errorclass = error_map.get(errno)\n        if errorclass is None:\n            errorclass = InternalError if errno < 1000 else OperationalError\n>       raise errorclass(errno, errval)\nE       pymysql.err.ProgrammingError: (1146, \"Table 'chengtay-zero-copy.zero_user_auth' doesn't exist\")\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pymysql\\err.py:107: ProgrammingError"}, "description": "ÊµãËØï http://10.10.128.152:10053/user/register ‰∏≠Â∞èÂæÆ‰ºÅ‰∏öÊ≥®ÂÜåÊµÅÁ®ã", "attachments": [{"name": "log", "source": "eea2753e-dd94-493e-aa23-46ac45eff07d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "64551a19-8d8f-4854-ba62-bb66b2205ad3-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "6ae6a158-a7b5-4e8f-9185-73f62e7e2d11-attachment.txt", "type": "text/plain"}], "start": 1601370359202, "stop": 1601370369258, "uuid": "547e10e0-2c34-4b56-a596-0141b56cef49", "historyId": "dd0646a5c1a798e47c4f25747509dea9", "testCaseId": "1fe2bac8daa9cfa6b5ae5069ce9a9205", "fullName": "test_flow.test_Authentication.test_login#test_companyRegister", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "A02-03-00033"}, {"name": "thread", "value": "11800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_login"}], "links": [{"type": "test_case", "url": "http://10.10.128.152:10053/user/register", "name": "Ê≥®ÂÜå üëá"}]}